ansible/roles/anitya-dev/tasks/main.yml

```diff
@@ -81,12 +81,12 @@
 - name: Install Anitya into the virtualenv
   become_user: "{{ ansible_env.SUDO_USER }}"
   pip:
-    name: "{{ item }}"
+    name: [
+            "file:///home/{{ ansible_env.SUDO_USER }}/devel",
+            "file:///home/{{ ansible_env.SUDO_USER }}/devel/anitya_schema"
+          ]
```

Comment:
Why use JSON syntax here instead of the way it was expressed previously?

Reply:
I used the same syntax that is used for dnf module in this ansible script.
The way it was expressed before is throwing deprecation warning and will not work from ansible 2.8 and above.

Reply:
i.e., this will also fix the deprecation warning, and won't be a mixture of YAML and JSON:

```
name:
  - "file:///home/{{ ansible_env.SUDO_USER }}/devel",
  - "file:///home/{{ ansible_env.SUDO_USER }}/devel/anitya_schema"
```

Reply:
Oh, I understand now. I used the same syntax as for dnf module in same ansible script.

It's confusing that the YAML supports also JSON format.

I will fix this.